examples/PyTorch/Inference/CUDA/ResNet/README.md

```diff
@@ -0,0 +1,56 @@
+# DISC optimization example for PyTorch ResNet model.
+
+The ResNet model is based on the [Deep Residual Learning for Image Recognition](https://arxiv.org/abs/1512.03385) paper.
+
+This repository uses the
+[model mantained in torchvision](https://pytorch.org/vision/stable/models/resnet.html).
+
+
+## Configure and run.
+
+These python packages are required for executing this example:
+
+- torch >= 1.6.0
+- torchvision
+- torch_blade
+
+To build and install `torch_blade` package, please refer to
+["Build BladeDISC from source"](../../../../docs/build_from_source.md) and
+["Install BladeDISC with Docker"](../../../../docs/install_with_docker.md).
+
+You can run the example with the following command:
+
+```bash
+python main.py
+```
+
+It will run optimizations of BladeDISC.
+
+## Performance results.
+
+We evaluate this example on A10 GPU, The CUDA version is 11.3. CuDNN version is
+8.3. PyTorch version is 1.12.0. The speedup reports on ResNet50 of the 200 inferences
+is as following.
+
+### `python main.py`
+|    | Backend   |   batch | precision   |   Median(FPS) |   Mean(FPS) |   Median(ms) |   Mean(ms) |    99th_p |     std_dev |
+|---:|:----------|--------:|:------------|--------------:|------------:|-------------:|-----------:|----------:|------------:|
+|  0 | eager     |      32 | fp32        |       930.02  |     930.859 |    0.0344079 |  0.0343776 | 0.0347254 | 0.000162988 |
+|  1 | script    |      32 | fp32        |       918.961 |     918.161 |    0.034822  |  0.0348532 | 0.0353194 | 0.000177631 |
```

Comment:
Does eager mean native PyTorch?
script is confused for users, maybe TorchScript is more clear.

Reply:
done